_posts/2018-11-12-sr2019-rules-released.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: news
+title: SR2019 Rules Released & Useful Resources
+---
+
+We had a successful Kickstart on Saturday and it was great to see all the excited teams getting familiar with our kit.
+
+Our game this year is called Caldera in which teams need to claim the most zones by placing more of their tokens than the enemy within the zone boundaries. ([full rules][rules])
+We’ve already seen the teams come up with some innovative solutions and it’s going to be interesting to see how they develop.
+
+All competitors should now have user accounts for the [IDE][ide]. (Please [contact us][contact] if you are not able to log in.)
+
+[Our forums][forums] for this year are now live. This is a great place to brainstorm with other teams and discuss any problems which you run into.
```

Comment:
Nice :)

Other things the forums can be used for (not that we need to include them all in the post -- just sharing for information and in case we do want to expand this):
- sharing links to a team's online activity -- blogs, videos etc. (relates to the Online Presence award)
- getting help from mentors (all mentors will get accounts shortly)